cmd/process-agent/app/subcommands/events/events.go

```diff
@@ -12,96 +12,110 @@ import (
 	"os"
 	"time"
 
+	"github.com/spf13/cobra"
+	"go.uber.org/fx"
+
```

Comment:
format: let's remove this blank line so we have 3 import groups instead of 4.

Reply:
Done!